.github/CODEOWNERS

```
@@ -0,0 +1,13 @@
+# This is used to auto-assign reviewers to pull requests, among other things.
+#
+# Note: Currently only works for users/teams with write access.
+# See: https://help.github.com/en/github/creating-cloning-and-archiving-reposoitories/about-code-owners
+
+*            @pol-is/core @pol-is/community
+
+/client-*/** @colinmegill
+
+/.github/**  @metasoarous
+/server/**   @metasoarous
+/math/**     @metasoarous
+Dockerfile*  @metasoarous
```

Could probably add me and @ballPointPenguin as stewards of cypress tests and docker stuff (though only me will get auto-added for now, oweing to my push access)

```suggestion
Dockerfile*  @metasoarous @patcon @ballpointpenguin
docker-compose.yml @metasoarous @patcon @ballpointpenguin

/e2e/**      @patcon @ballpointpenguin
```